assets/sass/components/_review-action-plan.scss

```diff
@@ -0,0 +1,5 @@
+.review-action-plan--bordered {
```

Comment:
A couple of thoughts on this:

- The `--` suffix is used for modifiers to an existing component; since we don't have a `review-action-plan` component to modify I don't think it makes sense. We should probably either call it `review-action-plan` or `review-action-plan-bordered`.
- The [suggested naming for custom components](https://design-system.service.gov.uk/get-started/extending-and-modifying-components/#use-a-unique-prefix-for-component-names) I think would suggest an `app-` prefix here.

I also wonder whether a [section break](https://design-system.service.gov.uk/styles/typography/#section-break) might be a more appropriate thing to use here? It _seems_ like we have a "thematic break between sections of content".

Reply:
I knew "section break" was a thing, but I couldn't find it in the GOV components section, didn't realise it was in Typography - thanks for the info, I think that's exactly what we need - have replaced it (also solves the issue of css naming)